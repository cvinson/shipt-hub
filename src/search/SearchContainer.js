import React from 'react';
import PropTypes from 'prop-types';
import Search from './Search';
import { getUser, getUserFollowers } from '../services/api';
import './Search.css';

const SearchContainer = ({ history, match }) => {
  return (
    <div className="searchContainer">
      <Search
        className="searchContainer"
        getUser={getUser}
        getUserFollowers={getUserFollowers}
        history={history}
        username={match.params.username} />
    </div>
  );
};

SearchContainer.propTypes = {
  history: PropTypes.object.isRequired,
  match: PropTypes.object.isRequired
}

export default SearchContainer;
