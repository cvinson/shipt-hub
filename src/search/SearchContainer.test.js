import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SearchContainer from './SearchContainer';

describe('Search', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<SearchContainer match={{ params: { username: 'githubUser' } }} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
