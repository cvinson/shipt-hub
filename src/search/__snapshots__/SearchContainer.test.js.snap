// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders correctly 1`] = `
<div
  className="searchContainer"
>
  <Search
    className="searchContainer"
    getUser={[Function]}
    getUserFollowers={[Function]}
    history={Object {}}
    username="githubUser"
  />
</div>
`;
